### DISCUSSION TITLE
"reason":"ERROR_UNAUTHORIZED_AMS_ENTITY"

### DISCUSSION THREAD

**Discussion by vipul-neon** [amzn/ads-advanced-tools-docs#20] [Q&A]
Hi, 
Why would I get this error ""ERROR_UNAUTHORIZED_AMS_ENTITY" only for select few campaigns if I am being able to make successful campaign API call for other sponsored brands' campaigns?
I have a valid access token, valid LWA client, and LWA secret too.

**Comment by ptaylor9999** [comment]
@vipul-neon You will receive that error message if you don't have permission to view those campaigns. Can you confirm that you have permission to view them?

**Reply by vipul-neon** [reply]
Hi, I am being able to see these campaigns on the console. However, I am not being able to access them via api

**Reply by ptaylor9999** [reply]
Thanks @vipul-neon - can you please confirm that the account you're using to access the console is the same account that's associated with your Login With Amazon application?

**Reply by vipul-neon** [reply]
thanks for your response. yes, i am using same login id

**Reply by ptaylor9999** [reply]
In that case you should have access to those accounts. Could you please go cut a support ticket at Amazon Ads API Service desk:

https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/user/login?destination=portal%2F2%2Fgroup%2F2

And include all the details of this thread. Thank you!
